src/table/virtualized-table/hooks/use-dynamic-row-height.ts

```diff
@@ -101,7 +105,7 @@ const useDynamicRowHeight = ({
         rowMeta.current.heights[rowIdx] = height;
       }
 
-      if (!batchStateUpdate) {
+      if (!batchStateUpdate && !isSnapshot) {
```

Comment:
I think you can move `!isSnapshot` to line 89. So that it will return early and not having to call `getCellSize` which is computationally heavy.

Reply:
Fixed.